python/jsbeautifier/javascript/beautifier.py

```diff
@@ -1184,13 +1184,21 @@ def handle_semicolon(self, current_token):
         self.print_token(current_token)
 
     def handle_string(self, current_token):
-        if self.start_of_statement(current_token):
+        if (
+            current_token.text[0]
+            == current_token.text[len(current_token.text) - 1]
+            == "`"
```

Comment:
I see here you skipped the regex.
```suggestion
            current_token.text[0]
            == "`"
```